gffquant/handle_args.py

```
@@ -58,23 +65,23 @@ def validate_args(args):
+        if args.run_mode != RunMode.GENE:
+            raise NotImplementedError("External gene counts currently require --mode genes.")
+
+    if args.strand_specific:
+        raise NotImplementedError("Strand-specific counting is currently disabled.")
+
     if args.restrict_metrics:
         restrict_metrics = set(args.restrict_metrics.split(","))
         invalid = restrict_metrics.difference(('raw', 'lnorm', 'scaled', 'rpkm'))
         if invalid:
-            raise ValueError(f"Invalid column(s) in `--restrict_metrics`: {str(invalid)}")
+            raise ValueError(f"Invalid metrics(s) in `--restrict_metrics`: {str(invalid)}")
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)